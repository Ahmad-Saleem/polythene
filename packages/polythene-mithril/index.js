export * from "polythene-mithril-base";
export * from "polythene-mithril-button";
export * from "polythene-mithril-fab";
export * from "polythene-mithril-icon";
export * from "polythene-mithril-raised-button";
export * from "polythene-mithril-ripple";
export * from "polythene-mithril-shadow";
export * from "polythene-mithril-svg";
