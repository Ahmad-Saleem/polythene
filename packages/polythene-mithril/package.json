{
  "name": "polythene-mithril",
  "version": "1.0.0-rc.1",
  "description": "",
  "main": "dist/polythene-mithril.js",
  "module": "dist/polythene-mithril.mjs",
  "scripts": {
    "clean": "rm -rf dist && mkdir dist",
    "rollup": "../../node_modules/rollup/bin/rollup -c ../../scripts/rollup.umd.js && ../../node_modules/rollup/bin/rollup -c ../../scripts/rollup.es.js",
    "build": "npm run rollup"
  },
  "files": [
    "dist",
    "README.md"
  ],
  "dependencies": {
    "polythene-mithril-base": "1.0.0-rc.1",
    "polythene-mithril-button": "1.0.0-rc.1",
    "polythene-mithril-card": "1.0.0-rc.1",
    "polythene-mithril-checkbox": "1.0.0-rc.1",
    "polythene-mithril-dialog": "1.0.0-rc.1",
    "polythene-mithril-dialog-pane": "1.0.0-rc.1",
    "polythene-mithril-fab": "1.0.0-rc.1",
    "polythene-mithril-icon": "1.0.0-rc.1",
    "polythene-mithril-icon-button": "1.0.0-rc.1",
    "polythene-mithril-list": "1.0.0-rc.1",
    "polythene-mithril-list-tile": "1.0.0-rc.1",
    "polythene-mithril-menu": "1.0.0-rc.1",
    "polythene-mithril-radio-button": "1.0.0-rc.1",
    "polythene-mithril-raised-button": "1.0.0-rc.1",
    "polythene-mithril-ripple": "1.0.0-rc.1",
    "polythene-mithril-shadow": "1.0.0-rc.1",
    "polythene-mithril-svg": "1.0.0-rc.1",
    "polythene-mithril-switch": "1.0.0-rc.1",
    "polythene-mithril-tabs": "1.0.0-rc.1",
    "polythene-mithril-textfield": "1.0.0-rc.1",
    "polythene-mithril-toolbar": "1.0.0-rc.1"
  },
  "author": "Arthur Clemens <arthurclemens@gmail.com> (http://visiblearea.com)",
  "license": "MIT"
}

