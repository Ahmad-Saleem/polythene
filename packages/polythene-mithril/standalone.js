export * from "./index";
import { addLayoutStyles, addFastClick } from "polythene-utilities";
import { addTypography, addRoboto } from "polythene-material-design";

addTypography();
addRoboto();
addLayoutStyles();
addFastClick();
