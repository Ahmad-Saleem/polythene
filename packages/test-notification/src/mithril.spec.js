import { runSnapshots } from "../../tests/scripts/mithril-snapshots";
import { renderer, keys, Notification, NotificationInstance } from "polythene-mithril";
import specTests from "./spec-tests.js";
import mithrilTests from "./tests-mithril.js";

runSnapshots({
  tests: specTests({ renderer, keys, Notification, NotificationInstance }).concat(mithrilTests),
  renderer
});
