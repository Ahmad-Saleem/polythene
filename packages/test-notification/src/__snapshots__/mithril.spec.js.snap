exports[`test No options - spec 1`] = `
"<div class=
\"pe-notification pe-dark-tone pe-notification--horizontal\">
  <div class=\"pe-notification__content\"></div>
</div>
"
`;

exports[`test Option: class - spec 1`] = `
"<div class=
\"pe-notification pe-dark-tone pe-notification--horizontal class-x\">
  <div class=\"pe-notification__content\"></div>
</div>
"
`;

exports[`test Option: className - spec 1`] = `
"<div class=
\"pe-notification pe-dark-tone pe-notification--horizontal className-x\">
<div class=\"pe-notification__content\"></div>
</div>
"
`;

exports[`test Option: element - spec 1`] = `
"<a class=
\"pe-notification pe-dark-tone pe-notification--horizontal\">
<div class=\"pe-notification__content\"></div></a>
"
`;

exports[`test Option: id - spec 1`] = `
"<div id=\"id-x\" class=
\"pe-notification pe-dark-tone pe-notification--horizontal\">
  <div class=\"pe-notification__content\"></div>
</div>
"
`;
